lib/fetch/index.js

```diff
@@ -126,8 +127,49 @@ class Fetch extends EE {
   }
 }
 
+// hoisting subscribers and context variables
+let subscribers
+
+// isSubscribed will check if their are subscribers to the channel
+// add set additional context before publishing the event
+function isSubscribed (name, context, type, load) {
```

Comment:
The name of this function is strange. from the name i would expect that the function is return a boolean function but in fact it is a publishIfSubscribed function?!

Reply:
I can change the name

Reply:
name changed